src/components/RefinementList/RefinementListItem.js

```diff
@@ -23,14 +23,14 @@ class RefinementListItem extends Component {
 
   render() {
     return (
-      <div className={this.props.itemClassName} onClick={this.handleClick}>
+      <li className={this.props.cssClasses.item} onClick={this.handleClick}>
         <Template
```

Comment:
This creates a wrapping `div` around the `label` from the template.

I can't think of an easy way to remove it because of the `subItems` that we want to render.

I don't think it's a big deal (we also have a wrapping `div` in the GeoSearch widget).

Reply:
does Preact not have Fragments?

Reply:
Nope, same issue with the GeoSearch widget (https://github.com/algolia/instantsearch.js/pull/3138#discussion_r219803316).

Reply:
Seems fine then :)